app/views/layouts/activities/email.html.slim

```diff
@@ -0,0 +1,10 @@
+h4
```

Comment:
Is this something specific to the activity mailer or something that should belong to the layout?